TCD-2913: Backtrace - last set of pages (#2292)
saucelabs/sauce-docs: docs/error-reporting/advanced/post-migration.md

---
id: post-migration
title: Post Migration Verification Steps
sidebar_label: Post migration
description: Required steps after a server migration.
---

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';
import useBaseUrl from '@docusaurus/useBaseUrl';

> After a server migration or similar work, the following are the minimum verification steps that should be completed.

Author: ```suggestion
After a server migration or similar work, the minimum verification steps need to be completed.
```

- Visit each config applet and make sure they all open correctly.

## Login/SSO

- Ensure that user accounts are brought over and able to login.
- If using SSO:
  - Verify that the configuration matches. Check if the option to configure SSO in the UI is present and does not return errors when navigated to.
  - Check if the **Login via SSO** button is available.
  - Test an SSO login.
  - Test an invitation request and confirm both the receipt of the email and the functioning of the link.

## Network/Endpoints/Certs

- Ensure that the server has the same listeners set up.
- Verify that the SSL config matches up on each SSL-configured listener.
- Make sure the proper DNS name is set up on the necessary endpoints.

## Misc. Services

- Verify that link sharing is functioning (if applicable).
- For each installed service:
  - Check its conf file in `/etc/backtrace/<servicename>/`.
  - Check for it in the morgue service status.
  - Verify that it's listening on its necessary port.

## Workflows/Reports

- Ensure that the workflow configurations are ported over.
- Verify that the workflows fire properly (check if the new server isn't having trouble talking to the third-party service).

> - Check the links within notifications.

Author: ```suggestion
- Check the links in notifications.
```
